arm-sql/2014-04-01/examples/ServerCreateMax.json

```diff
@@ -32,7 +32,7 @@
         "properties": {
           "fullyQualifiedDomainName": "sqlcrudtest-4645.database.windows.net",
           "administratorLogin": "dummylogin",
-          "administratorLoginPassword": "Un53cuRE!",
```

Comment:
Checked again, and we actually return the password for PUTs in 2014, but not 2015.